### PR TITLE
better rust support

### PR DESCRIPTION
**Pull request by stevefan1999-personal**

### CODE REVIEW

**Comment by MouriNaruto** [comment]
Note: RTM Build of Windows 7 is Build 7600. Build 9200 is Windows 8 RTM build.

Kenji Mouri

File: Rust/build.rs [hunk 1]
```diff
@@ -8,130 +8,108 @@
...
-    }
-    else if target_arch.contains("arm")
-    {
-        target_platform = "ARM";
-        if minimum_version < 9200
-        {
-            minimum_version = 9200;
+    let version = match std::env::var("VCLTL_VERSION") {
+        Ok(version) => version.parse::<usize>().unwrap(),
+        _ if cfg!(feature = "win7") => 9200,
```
Comment (mingkuang-Chuyu) [inline Rust/build.rs:46]: 9200 is Windows 8？

Comment (stevefan1999-personal) [reply Rust/build.rs:46]: I'm not sure either, so what's the actual value for Windows 7?

Comment (MouriNaruto) [reply Rust/build.rs:46]: Windows 7 RTM is Build 7600

Comment (mingkuang-Chuyu) [reply Rust/build.rs:46]: > I'm not sure either, so what's the actual value for Windows 7?

Windows 7 RTM is 7600?

**Comment by mingkuang-Chuyu** [comment]
Thanks for the PR, but I have some questions at the moment. 
1. How to solve the problem of 10MB cap of rust cates package?
2. The build version of Windows 7 RTM is 7600, not 9200.

**Comment by mingkuang-Chuyu** [comment]
We follow up on this issue from #63.

**Pull request closed without merging**